src/AutoMapper/QueryableExtensions/Impl/MemberGetterExpressionResultConverter.cs

```diff
@@ -21,7 +21,9 @@ public ExpressionResolutionResult GetExpressionResolutionResult(ExpressionResolu
         private static ExpressionResolutionResult ExpressionResolutionResult(
             ExpressionResolutionResult expressionResolutionResult, MemberInfo getter)
         {
-            var member = Expression.MakeMemberAccess(expressionResolutionResult.ResolutionExpression, getter);
+            var member = (getter is MethodInfo method && method.IsStatic && method.GetParameters().Count() == 1)
+                ? Expression.Call(method, expressionResolutionResult.ResolutionExpression) as Expression
```

Comment:
Consider just using a cast here instead of "as" which returns null if the value is not of that type (and we know that's impossible here).